docs/security-in-piwik.md

```diff
@@ -3,21 +3,6 @@ category: Develop
 ---
 # Security best practices
 
-## About this guide
```

Comment:
The "Introduction" section pretty much contained the same as the "About this guide" section but much shorter and more clear. Eg it does not say "if you want you can do this" it says "you must" which is better.